device.mk

```diff
@@ -244,7 +244,14 @@ PRODUCT_COPY_FILES += \
     device/asus/fugu/init.fugu.diag.rc.userdebug:root/init.fugu.diag.rc
 endif
 
-$(call inherit-product-if-exists, vendor/asus/fugu/device-vendor.mk)
+#$(call inherit-product-if-exists, vendor/asus/fugu/device-vendor.mk)
```

Comment:
device-vendor.mk installs AtvRemoteService.apk  LeanbackLauncher.apk  TV.apk. We don't need it.